cpp/test/longestConsecutiveSequence/longestConsecutiveSequenceTest.cpp

```diff
@@ -0,0 +1,44 @@
+#include "gtest/gtest.h"
```

Comment:
A good rule for arranging tests to add a line separator between arrange, act and assert sections

1. Arrange: where you arrange or create your input.
2. Act: When you call the procedure under test
3. Assert: when you are asserting the output.